templates/Web/Projects/VueDefault/package_postaction.json

```diff
@@ -0,0 +1,23 @@
+  },
+//{[{
+  "eslintConfig": {
```

Comment:
Here we insert parts of the package.json unique to vue. Defining eslint config + postcss.